Use double back-ticks in src/api/server/authn.rst

deepakchethan opened the pull request:
## Overview

As mentioned in #525, I have replaced the single back-ticks to double back-ticks **only in the file `src/api/server/authn.rst`**. I have modified all the instances which was resulting in italicisation. But I noticed that the single back-ticks have been used in a lot of other files.

Please let me know if all the files need to be updated. I'll update the pr with the same.

Thank you in advance!

## GitHub issue number

Fixes #525 

## Checklist

- [ ] Update [rebar.config.script](https://github.com/apache/couchdb/blob/main/rebar.config.script) with the commit hash once this PR is rebased and merged

nickva commented on src/api/server/authn.rst:
```diff
@@ -282,7 +282,7 @@ Proxy Authentication
         authentication_handlers = {chttpd_auth, cookie_authentication_handler}, {chttpd_auth, proxy_authentication_handler}, {chttpd_auth, default_authentication_handler}
 
-`Proxy authentication` is very useful in case your application already uses
+``Proxy authentication`` is very useful in case your application already uses
```
I am wondering ``...`` quote work better for literal code, config or API parameters. For section names or concepts italicization might read better?

```
`Proxy authentication` 
```
<img width="491" alt="Screen Shot 2021-07-07 at 2 00 14 PM" src="https://user-images.githubusercontent.com/211822/124807414-bc429500-df2b-11eb-90ab-da4689da603b.png">

vs
```
``Proxy authentication``
```
<img width="541" alt="Screen Shot 2021-07-07 at 2 00 22 PM" src="https://user-images.githubusercontent.com/211822/124807424-bea4ef00-df2b-11eb-85e7-a11d08e45abd.png">

Since ``Proxy authentication`` is not a config parameter.

The same thing applies to `Basic Auth`.

wohali replied:
This was intentionally italics. Not 100% of all instances of a single backtick should be replaced with double back-ticks.

nickva commented on src/api/server/authn.rst:
```diff
@@ -78,7 +78,7 @@ default, cookies are valid for 10 minutes, but it's :config:option:`adjustable
 
 To obtain the first token and thus authenticate a user for the first time, the
-`username` and `password` must be sent to the :ref:`_session API
+``username`` and ``password`` must be sent to the :ref:`_session API
```
Looks good as these are literal parameters we sent with the request

nickva commented:
Thanks for the contribution, @deepakchethan!

It's much appreciated. The password and username update I think looks great but I am not sure about non-literal stuff like section names or concepts. 

If you'd like find more literal parameters and as it would be great to get them all looking the same.

deepakchethan commented:
> Thanks for the contribution, @deepakchethan!
> 
> It's much appreciated. The password and username update I think looks great but I am not sure about non-literal stuff like section names or concepts.
> 
> If you'd like find more literal parameters and as it would be great to get them all looking the same.

Thank you for the review @nickva, I'll go over all the pages and try to update more literal parameters.

deepakchethan commented:
I have updated all the code/literal/config parameters as mentioned. I have also added these for headers as I saw it is being done in lot of places. Please let me know if I had missed/changed something that I shouldn't have. Thank you!

nickva commented:
Thank you, @deepakchethan! It's much appreciated!

 +1
